src/pretix/base/services/invoices.py

```diff
@@ -37,9 +35,6 @@
 @transaction.atomic
 def build_invoice(invoice: Invoice) -> Invoice:
     lp = invoice.order.payments.last()
-    open_payment = None
-    if lp and lp.state not in (OrderPayment.PAYMENT_STATE_CONFIRMED, OrderPayment.PAYMENT_STATE_REFUNDED):
-        open_payment = lp
```

Comment:
Hmhmm, interesting that we previously didn't call this at all for paid orders. However, since the default implementation in BasePaymentProvider and all implementations I just checked already take care of this situation, it should be fine.